src/__tests__/DrawerMenu.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import DrawerMenu from '../components/DrawerMenu';
+
+it('should render without breaking', () => {
+  const comp = shallow(<DrawerMenu id="0" to="" title="" nest={1} />);
+  expect(comp).toMatchSnapshot();
```

Comment:
I think the fact that I chose to write a snapshot test is the reason that this particular test throws a warning. Doing more research on the topic.